geode-redis/src/main/java/org/apache/geode/redis/internal/data/CommandHelper.java

```diff
@@ -137,38 +137,46 @@ private RedisString checkStringType(RedisData redisData, boolean ignoreTypeMisma
     return (RedisString) redisData;
   }
 
-  RedisString getRedisString(ByteArrayWrapper key) {
+  RedisString getRedisString(ByteArrayWrapper key, boolean affectsRedisStatistics) {
     RedisData redisData = getRedisData(key, NULL_REDIS_STRING);
+
     if (redisData == NULL_REDIS_STRING) {
-      redisStats.incKeyspaceMisses();
+      if (affectsRedisStatistics)
```

Comment:
Did you run spotless before submitting this PR? Please add a block for these single statements

Reply:
> I did run spotless. This code is not incorrect and I've seen this being used elsewhere in our codebase.

Reply:
`@bob` Just because Spotless didn't pick it up doesn't mean it's not part of our coding style. It makes for consistency and reduces the possibility of introducing future errors which I've seen happen when someone merges code that looks like this.

Reply:
> Why are you always nitpicking my code?